bodhi/validators.py

```diff
@@ -148,7 +148,7 @@ def validate_build_tags(request):
                 if not build_rel:
                     raise KeyError("Couldn't find release from build tags")
             except KeyError:
-                msg = 'Cannot find release associated with build: {}, tags: {}'.format(build, tags)
+                msg = 'Cannot find any tags associated with that nvr: {}, tags: {}'.format(build, tags)
```

Comment:
It would be nice to have the error messages be different. Maybe this could read something like 
```python
'nvr for {} is not tagged with any of the following: {}'.format(build, tags)
```